backend/server/routers/user.py

```diff
@@ -147,3 +147,46 @@ def update_degree_length(numYears: int, token: str = DUMMY_TOKEN):
     else:
         user['planner']['years'] = user['planner']['years'][:diff]
     set_user(token, user, True)
+
+@router.put("/setProgram") 
+def setProgram(token: str = DUMMY_TOKEN, degree: dict):
```

Comment:
if we're just using the `programCode` out of the degree dict, can have this accept that as a string instead